### PR TITLE
Bump `symfony/console` to `5.3.7` to support PHP 8.1

### LINKED ISSUE #246: PHP 8.1
I tried to use Pickle on PHP 8.1 RC6 but I'm getting a lot of errors:

```
pickle -v
PHP Deprecated:  Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Helper/HelperSet.php on line 94
PHP Stack trace:
PHP   1. {main}() /Volumes/CS/www/website/var/pickle.phar:0
PHP   2. require() /Volumes/CS/www/website/var/pickle.phar:10
PHP   3. Symfony\Component\Console\Application->run($input = *uninitialized*, $output = *uninitialized*) phar:///Volumes/CS/www/website/var/pickle.phar/bin/pickle:38
PHP   4. Symfony\Component\Console\Application->doRun($input = class Symfony\Component\Console\Input\ArgvInput { protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['verbose' => TRUE]; protected $arguments = []; protected $interactive = TRUE; private $tokens = [0 => '-v']; private $parsed = [] }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(74) of type (stream); private $stderr = class Symfony\Component\Console\Output\StreamOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(89) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:166
PHP   5. Symfony\Component\Console\Application->find($name = 'list') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:254
PHP   6. Symfony\Component\Console\Application->init() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:628
PHP   7. Symfony\Component\Console\Application->add($command = class Symfony\Component\Console\Command\HelpCommand { private ${Symfony\Component\Console\Command\Command}application = class Pickle\Console\Application { private ${Symfony\Component\Console\Application}commands = [...]; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = NULL; private ${Symfony\Component\Console\Application}name = '\n ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗\n ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝\n ██████╔╝██║██║     █████╔╝ ██║     █████╗\n ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝\n ██║     ██║╚██████╗██║  ██╗�'; private ${Symfony\Component\Console\Application}version = 'v0.7.7'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { ... }; private ${Symfony\Component\Console\Application}helperSet = NULL; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal { ... }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE; private ${Symfony\Component\Console\Application}signalRegistry = class Symfony\Component\Console\SignalRegistry\SignalRegistry { ... }; private ${Symfony\Component\Console\Application}signalsToDispatchEvent = [...] }; private ${Symfony\Component\Console\Command\Command}name = 'help'; private ${Symfony\Component\Console\Command\Command}processTitle = NULL; private ${Symfony\Component\Console\Command\Command}aliases = []; private ${Symfony\Component\Console\Command\Command}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}hidden = FALSE; private ${Symfony\Component\Console\Command\Command}help = 'The <info>%command.name%</info> command displays help for a given command:\n\n  <info>%command.full_name% list</info>\n\nYou can also output the help in other formats by using the <comment>--format</comment> option:\n\n  <info>%command.full_name% --format=xml list</info>\n\nTo display the list of available commands, please use the <info>list</info> command.'; private ${Symfony\Component\Console\Command\Command}description = 'Displays help for a command'; private ${Symfony\Component\Console\Command\Command}fullDefinition = NULL; private ${Symfony\Component\Console\Command\Command}ignoreValidationErrors = TRUE; private ${Symfony\Component\Console\Command\Command}code = NULL; private ${Symfony\Component\Console\Command\Command}synopsis = []; private ${Symfony\Component\Console\Command\Command}usages = []; private ${Symfony\Component\Console\Command\Command}helperSet = NULL; private $command = NULL }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:1230
PHP   8. Symfony\Component\Console\Command\Command->setApplication($application = class Pickle\Console\Application { private ${Symfony\Component\Console\Application}commands = []; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = NULL; private ${Symfony\Component\Console\Application}name = '\n ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗\n ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝\n ██████╔╝██║██║     █████╔╝ ██║     █████╗\n ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝\n ██║     ██║╚██████╗██║  ██╗�'; private ${Symfony\Component\Console\Application}version = 'v0.7.7'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Application}helperSet = NULL; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal {  }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE; private ${Symfony\Component\Console\Application}signalRegistry = class Symfony\Component\Console\SignalRegistry\SignalRegistry { private $signalHandlers = [...] }; private ${Symfony\Component\Console\Application}signalsToDispatchEvent = [0 => 2, 1 => 15, 2 => 30, 3 => 31] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:484
PHP   9. Symfony\Component\Console\Application->getHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Command/Command.php:96
PHP  10. Pickle\Console\Application->getDefaultHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:316
PHP  11. Symfony\Component\Console\Application->getDefaultHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/src/Console/Application.php:58
PHP  12. Composer\Autoload\ClassLoader->loadClass($class = 'Symfony\\Component\\Console\\Helper\\HelperSet') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:1059
PHP  13. Composer\Autoload\includeFile($file = 'phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/../symfony/console/Helper/HelperSet.php') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/ClassLoader.php:346
PHP  14. include() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/ClassLoader.php:480

Deprecated: Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Helper/HelperSet.php on line 94

Call Stack:
    0.0071    3211904   1. {main}() /Volumes/CS/www/website/var/pickle.phar:0
    0.0071    3208512   2. require('phar:///Volumes/CS/www/website/var/pickle.phar/bin/pickle') /Volumes/CS/www/website/var/pickle.phar:10
    0.0103    3773640   3. Symfony\Component\Console\Application->run($input = ???, $output = ???) phar:///Volumes/CS/www/website/var/pickle.phar/bin/pickle:38
    0.0218    4072008   4. Symfony\Component\Console\Application->doRun($input = class Symfony\Component\Console\Input\ArgvInput { protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['verbose' => TRUE]; protected $arguments = []; protected $interactive = TRUE; private $tokens = [0 => '-v']; private $parsed = [] }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(74) of type (stream); private $stderr = class Symfony\Component\Console\Output\StreamOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(89) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:166
    0.0221    4166032   5. Symfony\Component\Console\Application->find($name = 'list') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:254
    0.0221    4166032   6. Symfony\Component\Console\Application->init() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:628
    0.0238    4354112   7. Symfony\Component\Console\Application->add($command = class Symfony\Component\Console\Command\HelpCommand { private ${Symfony\Component\Console\Command\Command}application = class Pickle\Console\Application { private ${Symfony\Component\Console\Application}commands = [...]; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = NULL; private ${Symfony\Component\Console\Application}name = '\n ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗\n ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝\n ██████╔╝██║██║     █████╔╝ ██║     █████╗\n ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝\n ██║     ██║╚██████╗██║  ██╗�'; private ${Symfony\Component\Console\Application}version = 'v0.7.7'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { ... }; private ${Symfony\Component\Console\Application}helperSet = NULL; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal { ... }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE; private ${Symfony\Component\Console\Application}signalRegistry = class Symfony\Component\Console\SignalRegistry\SignalRegistry { ... }; private ${Symfony\Component\Console\Application}signalsToDispatchEvent = [...] }; private ${Symfony\Component\Console\Command\Command}name = 'help'; private ${Symfony\Component\Console\Command\Command}processTitle = NULL; private ${Symfony\Component\Console\Command\Command}aliases = []; private ${Symfony\Component\Console\Command\Command}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}hidden = FALSE; private ${Symfony\Component\Console\Command\Command}help = 'The <info>%command.name%</info> command displays help for a given command:\n\n  <info>%command.full_name% list</info>\n\nYou can also output the help in other formats by using the <comment>--format</comment> option:\n\n  <info>%command.full_name% --format=xml list</info>\n\nTo display the list of available commands, please use the <info>list</info> command.'; private ${Symfony\Component\Console\Command\Command}description = 'Displays help for a command'; private ${Symfony\Component\Console\Command\Command}fullDefinition = NULL; private ${Symfony\Component\Console\Command\Command}ignoreValidationErrors = TRUE; private ${Symfony\Component\Console\Command\Command}code = NULL; private ${Symfony\Component\Console\Command\Command}synopsis = []; private ${Symfony\Component\Console\Command\Command}usages = []; private ${Symfony\Component\Console\Command\Command}helperSet = NULL; private $command = NULL }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:1230
    0.0238    4354112   8. Symfony\Component\Console\Command\Command->setApplication($application = class Pickle\Console\Application { private ${Symfony\Component\Console\Application}commands = []; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = NULL; private ${Symfony\Component\Console\Application}name = '\n ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗\n ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝\n ██████╔╝██║██║     █████╔╝ ██║     █████╗\n ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝\n ██║     ██║╚██████╗██║  ██╗�'; private ${Symfony\Component\Console\Application}version = 'v0.7.7'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Application}helperSet = NULL; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal {  }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE; private ${Symfony\Component\Console\Application}signalRegistry = class Symfony\Component\Console\SignalRegistry\SignalRegistry { private $signalHandlers = [...] }; private ${Symfony\Component\Console\Application}signalsToDispatchEvent = [0 => 2, 1 => 15, 2 => 30, 3 => 31] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:484
    0.0238    4354112   9. Symfony\Component\Console\Application->getHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Command/Command.php:96
    0.0238    4354112  10. Pickle\Console\Application->getDefaultHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:316
    0.0238    4354112  11. Symfony\Component\Console\Application->getDefaultHelperSet() phar:///Volumes/CS/www/website/var/pickle.phar/src/Console/Application.php:58
    0.0238    4354112  12. Composer\Autoload\ClassLoader->loadClass($class = 'Symfony\\Component\\Console\\Helper\\HelperSet') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:1059
    0.0238    4354112  13. Composer\Autoload\includeFile($file = 'phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/../symfony/console/Helper/HelperSet.php') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/ClassLoader.php:346
    0.0240    4360808  14. include('phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Helper/HelperSet.php') phar:///Volumes/CS/www/website/var/pickle.phar/vendor/composer/ClassLoader.php:480


 ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗
 ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝
 ██████╔╝██║██║     █████╔╝ ██║     █████╗
 ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝
 ██║     ██║╚██████╗██║  ██╗███████╗███████╗
 ╚═╝     ╚═╝ ╚═════╝╚═╝  ╚═╝╚══════╝╚══════╝  v0.7.7

Usage:
  command [options] [arguments]

PHP Deprecated:  mb_detect_encoding(): Passing null to parameter #1 ($string) of type string is deprecated in phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Helper/Helper.php on line 48
PHP Stack trace:
PHP   1. {main}() /Volumes/CS/www/website/var/pickle.phar:0
PHP   2. require() /Volumes/CS/www/website/var/pickle.phar:10
PHP   3. Symfony\Component\Console\Application->run($input = *uninitialized*, $output = *uninitialized*) phar:///Volumes/CS/www/website/var/pickle.phar/bin/pickle:38
PHP   4. Symfony\Component\Console\Application->doRun($input = class Symfony\Component\Console\Input\ArgvInput { protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['verbose' => TRUE]; protected $arguments = []; protected $interactive = TRUE; private $tokens = [0 => '-v']; private $parsed = [] }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(74) of type (stream); private $stderr = class Symfony\Component\Console\Output\StreamOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(89) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:166
PHP   5. Symfony\Component\Console\Application->doRunCommand($command = class Symfony\Component\Console\Command\ListCommand { private ${Symfony\Component\Console\Command\Command}application = class Pickle\Console\Application { private ${Symfony\Component\Console\Application}commands = [...]; private ${Symfony\Component\Console\Application}wantHelps = FALSE; private ${Symfony\Component\Console\Application}runningCommand = ...; private ${Symfony\Component\Console\Application}name = '\n ██████╗ ██╗ ██████╗██╗  ██╗██╗     ███████╗\n ██╔══██╗██║██╔════╝██║ ██╔╝██║     ██╔════╝\n ██████╔╝██║██║     █████╔╝ ██║     █████╗\n ██╔═══╝ ██║██║     ██╔═██╗ ██║     ██╔══╝\n ██║     ██║╚██████╗██║  ██╗�'; private ${Symfony\Component\Console\Application}version = 'v0.7.7'; private ${Symfony\Component\Console\Application}commandLoader = NULL; private ${Symfony\Component\Console\Application}catchExceptions = TRUE; private ${Symfony\Component\Console\Application}autoExit = TRUE; private ${Symfony\Component\Console\Application}definition = class Symfony\Component\Console\Input\InputDefinition { ... }; private ${Symfony\Component\Console\Application}helperSet = class Symfony\Component\Console\Helper\HelperSet { ... }; private ${Symfony\Component\Console\Application}dispatcher = NULL; private ${Symfony\Component\Console\Application}terminal = class Symfony\Component\Console\Terminal { ... }; private ${Symfony\Component\Console\Application}defaultCommand = 'list'; private ${Symfony\Component\Console\Application}singleCommand = FALSE; private ${Symfony\Component\Console\Application}initialized = TRUE; private ${Symfony\Component\Console\Application}signalRegistry = class Symfony\Component\Console\SignalRegistry\SignalRegistry { ... }; private ${Symfony\Component\Console\Application}signalsToDispatchEvent = [...] }; private ${Symfony\Component\Console\Command\Command}name = 'list'; private ${Symfony\Component\Console\Command\Command}processTitle = NULL; private ${Symfony\Component\Console\Command\Command}aliases = []; private ${Symfony\Component\Console\Command\Command}definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}hidden = FALSE; private ${Symfony\Component\Console\Command\Command}help = 'The <info>%command.name%</info> command lists all commands:\n\n  <info>%command.full_name%</info>\n\nYou can also display the commands for a specific namespace:\n\n  <info>%command.full_name% test</info>\n\nYou can also output the information in other formats by using the <comment>--format</comment> option:\n\n  <info>%command.full_name% --format=xml</info>\n\nIt\'s also possible to get raw list of commands (useful for embedding command runner):\n\n  <info>%command.full_name% --raw</info>'; private ${Symfony\Component\Console\Command\Command}description = 'Lists commands'; private ${Symfony\Component\Console\Command\Command}fullDefinition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; private ${Symfony\Component\Console\Command\Command}ignoreValidationErrors = FALSE; private ${Symfony\Component\Console\Command\Command}code = NULL; private ${Symfony\Component\Console\Command\Command}synopsis = []; private ${Symfony\Component\Console\Command\Command}usages = []; private ${Symfony\Component\Console\Command\Command}helperSet = class Symfony\Component\Console\Helper\HelperSet { private $helpers = [...]; private $command = NULL } }, $input = class Symfony\Component\Console\Input\ArgvInput { protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['verbose' => TRUE]; protected $arguments = []; protected $interactive = TRUE; private $tokens = [0 => '-v']; private $parsed = [] }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(74) of type (stream); private $stderr = class Symfony\Component\Console\Output\StreamOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(89) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:290
PHP   6. Symfony\Component\Console\Command\Command->run($input = class Symfony\Component\Console\Input\ArgvInput { protected $definition = class Symfony\Component\Console\Input\InputDefinition { private $arguments = [...]; private $requiredCount = 0; private $hasAnArrayArgument = FALSE; private $hasOptional = TRUE; private $options = [...]; private $shortcuts = [...] }; protected $stream = NULL; protected $options = ['verbose' => TRUE]; protected $arguments = []; protected $interactive = TRUE; private $tokens = [0 => '-v']; private $parsed = [] }, $output = class Symfony\Component\Console\Output\ConsoleOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { private $decorated = TRUE; private $styles = [...]; private $styleStack = class Symfony\Component\Console\Formatter\OutputFormatterStyleStack { ... } }; private ${Symfony\Component\Console\Output\StreamOutput}stream = resource(74) of type (stream); private $stderr = class Symfony\Component\Console\Output\StreamOutput { private ${Symfony\Component\Console\Output\Output}verbosity = 64; private ${Symfony\Component\Console\Output\Output}formatter = class Symfony\Component\Console\Formatter\OutputFormatter { ... }; private $stream = resource(89) of type (stream) }; private $consoleSectionOutputs = [] }) phar:///Volumes/CS/www/website/var/pickle.phar/vendor/symfony/console/Application.php:971
```

### PR DESCRIPTION
**Pull request by ruudk**
Fixes #246

### CODE REVIEW

**Comment by mlocati** [comment]
I'd update the composer.lock file instead of composer.json (that is, simply run `composer update symfony/console`)

**Comment by ruudk** [comment]
I'm sorry, I wasn't aware the lock file was committed. Will fix it :)

**Comment by ruudk** [comment]
@mlocati Ready :)

**Comment by ruudk** [comment]
@mlocati Anything needed to get this merged? 🙏

**Comment by ruudk** [comment]
@llaville Updated the PR to use `symfony/console` `5.4.1`, see https://github.com/FriendsOfPHP/pickle/issues/246#issuecomment-997410504

**Comment by llaville** [comment]
Minimun requirement for `symfony/console` v5 is trully **5.3.7**. Confimed by origin of change : see https://github.com/symfony/console/commit/9a4fcecbcccf4b55f4c28e1c71e3cd83d7be99b8

**Comment by ruudk** [comment]
@pierrejoye What's needed to get this merged?

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "justinrainbow/json-schema": "^3.0 || ^4.0 || ^5.0",
         "mlocati/composer-patcher": "^1.2.2",
         "padraic/phar-updater": "~1.0@dev",
-        "symfony/console": "^5.0"
+        "symfony/console": "^5.3.7"
```
Comment (mlocati) [inline composer.json:30]: Why did you change this line? Does this project really needs at least version 5.3.7?

**Comment by mlocati** [comment]
> What's needed to get this merged?

I think a person that has some spare time to maintain this project :wink:

File: composer.json (hunk 1 above)
Comment (ruudk) [reply composer.json:30]: Yes, because PHP 8.1 support is added from that point.

**Comment by ruudk** [comment]
@mlocati 🙋

File: composer.json (hunk 1 above)
Comment (mlocati) [reply composer.json:30]: > because PHP 8.1 support is added from that point.

That's correct. But it's enough to have 5.3.7+ in composer.lock.
Having ^5.3.7 in composer.json forces other projects that include pickle to have this requirement.
If pickle doesn't use features included in 5.3.7+ features, it's enough to have ^5.0

Comment (ruudk) [reply composer.json:30]: Actually, I think the tests should test `lowest` and `highest` dependencies and not rely on `composer.lock`.

Comment (ruudk) [reply composer.json:30]: Also, the docs only mention to install pickle with the phar...

Comment (ruudk) [reply composer.json:30]: So it could even become Symfony 5.4

Comment (mlocati) [reply composer.json:30]: > the docs only mention to install pickle with the phar

Sure, but since pickle is published on packagist, people may use its features in their projects too.

**Comment by mlocati** [comment]
> @mlocati 🙋

I'm not the official maintainer, I can't take any decision about that.

**Comment by mlocati** [comment]
In #250 I'm updating symfony/console (and all the other dependencies)
